emscripten.py

```diff
@@ -1887,9 +1885,13 @@ def create_runtime_funcs_asmjs(exports, metadata):
 }
 ''' % stack_check]
 
+  if shared.Settings.MINIMAL_RUNTIME:
+    # MINIMAL_RUNTIME moves stack functions to library.
+    funcs = []
+
   if shared.Settings.USE_PTHREADS:
     funcs.append('''
-function establishStackSpace(stackBase, stackMax) {
+function asmJsEstablishStackFrame(stackBase, stackMax) {
```

Comment:
Renamed this to highlight the fact that this functionality is only needed for old fastcomp threading support.